### PR TITLE
Create new Button component

### PR DESCRIPTION
**Pull request by jufglanville**
- Jira Ticket: [MEN-6](https://jufglanvlle.atlassian.net/browse/MEN-6?atlOrigin=eyJpIjoiYzI4YjVlNjJlNGNmNDczZTlmZmMwZmI2ZWZlOGFhYTMiLCJwIjoiaiJ9)

- Create a new Button component, this takes two props, a `type` this can be either `button` or `cancel` and affects styles. This also accepts the `children` prop

### CODE REVIEW

File: src/components/Button.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import styled from 'styled-components';
+
+interface Props {
+  type: 'button' | 'cancel';
```
Comment (JakeStaTeresa) [inline src/components/Button.tsx]: I think this is a good approach. But I would make it primary | button.

File: src/components/Button.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import styled from 'styled-components';
+
+interface Props {
```
Comment (JakeStaTeresa) [inline src/components/Button.tsx:8]: Would be nice to have a click handler here

**Review by JakeStaTeresa — CHANGES_REQUESTED** [review]

File: src/components/Button.tsx (hunk 1 above)
Comment (jufglanville) [reply src/components/Button.tsx]: This has been updated to be `primary | button` option types

File: src/components/Button.tsx (hunk 2 above)
Comment (jufglanville) [reply src/components/Button.tsx:8]: I have added a simple click handler that currently runs a `handleClick` event in the parent `SummaryCard` component and simply console logs a message

Comment (JakeStaTeresa) [reply src/components/Button.tsx:8]: In terms of naming, I would start the events with the prefix on. handleClick would be how I would name a function that is assigned to onClick.

**Review by JakeStaTeresa — APPROVED** [review]

**Pull request closed without merging**